src/pages/overlay/hooks/useSettings.tsx

```diff
@@ -33,6 +33,21 @@ const settings = {
       ),
     configurable: true,
   },
+  ambassadorSort: {
+    title: "Sort ambassadors by",
+    type: "select",
+    options: [
+      { value: "default", label: "Default" },
+      { value: "arrival", label: "Arrival date" },
+      { value: "name", label: "Name (A-Z)" },
+      { value: "species", label: "Species (A-Z)" },
+    ],
```

Comment:
Can we move this options stanza into a new `utils/sorting.ts` file as `sortOptions`, and then use that to generate a `sortMethods` array + a `SortMethod` type? So that we're defining these once instead of repeatedly throughout this PR?